ZitiUpdateService/checkers/GithubCheck.cs

```diff
@@ -38,7 +45,7 @@ internal class GithubCheck : UpdateCheck {
 			return downloadFileName;
 		}
 
-		override public int IsUpdateAvailable(Version currentVersion) {
+		override public void IsUpdateAvailable(Version currentVersion, out int avail, out string publishedDate) {

```

Comment:
small nit: Maybe change the function name to reflect more accurately the purpose... just "CheckUpdate" or something?